src/h2load.cc

```diff
@@ -352,6 +414,16 @@ Client::Client(uint32_t id, Worker *worker, size_t req_todo)
 
   ev_timer_init(&request_timeout_watcher, client_request_timeout_cb, 0., 0.);
   request_timeout_watcher.data = this;
+
+  if (worker->config->warm_up_time > 0) {
+    ev_timer_init(&duration_watcher, duration_timeout_cb, 
+                  worker->config->rate_period, 0.);
```

Comment:
I think you should switch away from using `rate_period`, it's a bit confusing, since it is being used here as a benchmark duration but in other scenarios as an actual rate.